src/service-object.ts

```diff
@@ -573,6 +590,6 @@ class ServiceObject<T = any> extends EventEmitter {
   }
 }
 
-promisifyAll(ServiceObject);
+promisifyAll(ServiceObject, {exclude: ['getRequestInterceptors']});
```

Comment:
Will this have an impact on downstream consumers of the library?  Or example - let's say in Storage, I inherit from `ServiceObject`, and then call `promisifyAll(Bucket)`.  Would I then need to specify the same list of methods to exclude, or does having it here cover me?  Part of this is that I don't remember if `promisifyAll` operates over inherited methods, or just own properties.

Reply:
>  Would I then need to specify the same list of methods to exclude

Nope, it's covered here. It's only working immediately on the methods from the class.